pvlib/atmosphere.py

```diff
@@ -467,3 +467,170 @@ def first_solar_spectral_correction(pw, airmass_absolute, module_type=None,
         coeff[4]*np.sqrt(pw) + coeff[5]*ama/np.sqrt(pw))
 
     return modifier
+
+
+def kasten96_lt(aod, am, pwat, alpha0=1.14, method='Molineaux'):
+    """
+    Calculate Linke turbidity factor using Kasten pyrheliometric formula (1980).
+
+    Aerosol optical depth can be given as a list of tuples with wavelength in
+    nanometers as the first item in each tuple and values as AOD as the second
+    item. The list must be in ascending order by wavelength. If ``aod`` is given
+    as a sequence of floats or as a float then a wavelength of 500[nm] will be
+    used and alpha will default to 1.14, unless alpha is also given. Otherwise
+    alpha is calculated from the given wavelength and aod.
+
+    Method can be either ``'Molineaux'`` or ``'Bird-Huldstrom'``. Airmass less
+    than 1 or greater than 6 will return ``NaN``. Precipitable water less than
+    zero or greater than 5[cm] will also return ``NaN``.
+
+    Based on originam implementation by Armel Oumbe.
+
+    Parameters
+    ----------
+    aod : numeric
+        aerosol optical depth table or value at 500
+    am : numeric
+        airmass, pressure corrected in atmospheres
+    pwat : numeric
+        precipitable water or total column water vapor in centimeters
+    alpha0 : numeric
+        Angstrom turbidity alpha exponent, default is 1.14
+    method : str
+        Molineaux (default) or Bird-Huldstrom
+
+    Returns
+    -------
+        Linke turbidity
+
+    References
+    ----------
```

Comment:
I fixed the references errors. Not sure I agree about the references. It's hard for me to tease out all of these pieces from just one or two references. I suppose we could just use the Ineichen references which include almost all of the other ones, although some are incorrect:
* "A new airmass independent ..." (2002) includes: Kasten-1980, Kasten-1996, Linke-1922, Molineaux-1989
* "Conversion function between Linke ..." (2008) includes: Berk-1989,1996, Bird-1980 (incorrect reference), Mueller and Mayer

What is your preference?